src/init/update-artifacts/test/exampleTest.js

```diff
@@ -4,11 +4,11 @@ const { utils, wallets } = require('@aeternity/aeproject');
 const EXAMPLE_CONTRACT_SOURCE = './contracts/ExampleContract.aes';
 
 describe('ExampleContract', () => {
-  let client;
+  let aeSdk;
   let contract;
 
   before(async () => {
-    client = await utils.getClient();
+    aeSdk = await utils.getSdk();
```

Comment:
technically incorrect, its an initialized instance of the sdk we are getting, not the sdk itself

Reply:
I'd really like to not discuss this until 2023. back then I got almost no response in https://github.com/aeternity/aepp-sdk-js/issues/1300

would you be fine with `aeSdk = await utils.getSdkInstance();` ?

Reply:
though in the docs we describe it already like this:

- `getSdk()` get an instance of the SDK for local development
        - initialized with all prefunded wallets for `onAccount` to be used calling from different accounts

Reply:
I really wanna avoid discussing `aeSdk`

Reply:
> I really wanna avoid discussing `aeSdk`

that was my feedback all along, but I guess its not my decision

Reply:
I heard it the first time some weeks ago and I didn't notice any other comment of your side back then when we discussed that. if you told this in some chat I just cannot remember.

now - at least for the moment - I'd be happy if we can go this route.

maybe `client` is technically more correct. but it's also not explicitly telling us what it's about (at least right now).

I totally get your point, but I don't see it that strict. I would propose that we use the governance aepp in the future if our dev community gets bigger to vote on this 😅 I'd actually like such vote. but currently it wouldn't make sense IMO.